Sofa/Component/Collision/Detection/Algorithm/src/sofa/component/collision/detection/algorithm/CollisionPipeline.h

```diff
@@ -58,7 +58,7 @@ class SOFA_COMPONENT_COLLISION_DETECTION_ALGORITHM_API CollisionPipeline : publi
 };
 
 
-SOFA_DEPRECATED_HEADER("v23.06", "v23.12", "sofa/component/collision/detection/algorithm/CollisionPipeline.h")
+SOFA_ATTRIBUTE_DISABLED__RENAME_COLLISIONPIPELINE()
 using DefaultPipeline = sofa::component::collision::detection::algorithm::CollisionPipeline;
```

Comment:
> ```c++
> typedef DeprecatedAndRemoved DefaultPipeline
> ```
> 
> The doc says:
> 
>     * add a C++ alias for the component name
> 
>     * add a macro SOFA_ATTRIBUTE_DEPRECATED on this alias
> 

`  typedef DeprecatedAndRemoved DefaultPipeline` 
this has the effect to directly remove the use of DefaultPipeline in C++ (as it will be aliased to something not usable)
(in case of, I shall remind you that `using`  and ` typedef` are mostly the same, i.e creating aliases; `using` being recommended because it can handle templates)

In my mind what has been done until now is following the guidelines 😗

Reply:
yes @hugtalbot